app/controllers/sessions_controller.rb

```diff
@@ -1,11 +1,28 @@
 class SessionsController < ApplicationController
-	layout "application"
+    layout "application"
 
 	def show
 		if user_signed_in?
 			user = current_user
 		end
 		session = Session.find(params[:id])
 		@session_props = {session: session, user: user}
-	end
+    end
+    
+    def create
+        pt = PriceTable.find_by(price_point: session_params[:price_point])
+        @session = Session.new(session_params.except(:price_point).merge(price_table_id: pt.id))
```

Comment:
Why are we doing this?

Reply:
Because we need to have the price table when we are creating a new session